nrrd/writer.py

```diff
@@ -121,6 +121,9 @@ def write(filename, data, header={}, detached_header=False, custom_field_map=Non
         Data to save to the NRRD file
     detached_header : :obj:`bool`, optional
         Whether the header and data should be saved in separate files. Defaults to :obj:`False`
+    relative_data_path : :class:`bool`
+        Whether the data file name in detached header is saved with a relative path or absolute path.
```

Comment:
Alright, rereading this now I agree with how it's set.

Just one minor thing, I usually prefer the use of `filename` rather than `file name`.  To stay consistent let's change it. I think I have it without the space in the docs and such.
```suggestion
        Whether the data filename in detached header is saved with a relative path or absolute path.
```